test/functional/specs/C10922.js

```diff
@@ -0,0 +1,171 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { t, ClientFunction } from "testcafe";
+import createNetworkLogger from "../helpers/networkLogger";
+import fixtureFactory from "../helpers/fixtureFactory";
+import configureAlloyInstance from "../helpers/configureAlloyInstance";
+import {
+  compose,
+  edgeDomainThirdParty,
+  edgeDomainFirstParty,
+  orgMainConfigMain,
+  thirdPartyCookiesDisabled,
+  thirdPartyCookiesEnabled,
+  migrationEnabled,
+  migrationDisabled
+} from "../helpers/constants/configParts";
+import reloadPage from "../helpers/reloadPage";
+import setLegacyIdentityCookie from "../helpers/setLegacyIdentityCookie";
+
+const networkLogger = createNetworkLogger();
+
+const executeEventCommand = ClientFunction(() => {
+  return window.alloy("event");
+});
+
+const idThirdPartyDomainRegex = /\.demdex\.net/;
+
+const getHostForFirstRequest = () => {
+  const firstRequest = networkLogger.edgeInteractEndpointLogs.requests[0];
+  return firstRequest.request.headers.host;
+};
+
+const assertRequestWentToIdThirdPartyDomain = async () => {
```

Comment:
All these "third party" names hurt my head.  Do you think this would be easier to understand if we just said "Demdex". (i.e. `assertRequestWentToDemdex`)

Reply:
I was hoping to keep the name brand-agnostic and more role-oriented but, yes, it would probably be easier to understand with "demdex". I'll change it.